site_root/input/resources/examples/Task-example-research-study-task-id.json

```diff
@@ -0,0 +1,34 @@
+{
+  "resourceType": "Task",
+  "id": "example-research-study-task-id",
+  "status": "accepted",
+  "intent": "unknown",
+  "input": [
+    {
+      "type": {
+        "coding": [
+          {
+            "code": "ResearchStudy"
+          }
+        ]
+      },
+      "valueReference": {
+        "reference": "ResearchStudy/example-project-id"
+      }
+    }
```

Comment:
I am not sure whether a ResearchStudy could be an input of Task. `Specimen` could be like https://github.com/kids-first/kf-model-fhir/blob/add-se-conf-resources/site_root/input/resources/examples/Task-st-001.json

Reply:
resolved - implemented.